badge-maker/lib/color.js

```diff
@@ -4,15 +4,15 @@ const { fromString } = require('css-color-converter')
 
 // When updating these, be sure also to update the list in `badge-maker/README.md`.
```

Comment:
In the PR description you did mention:

> This PR makes changes to the badge-maker NPM package. I haven't bumped the package version or updated the changelog in this PR.

so just wanted to add the readme file to that list of "to be updated" given this comment